### ISSUE TITLE
Parse killmails

### ISSUE THREAD

**Issue by karbowiak** [EVE-KILL/projectRena#48]
To parse killmails several things has to be done first..

New style / CREST / API formatted killmails that are already being dropped into the database without any issues.
- [x] Point calculator function (Calculate points based on ship killed value and whatnots, just lift it from zKB)
- [x] Market data population (EVE Central?)
- [x] Victim Parser (Taking data from json and formatting into data for the participants table)
- [x] Attacker Parser (Taking data from json and formatting into data for the participants table)
- [x] Item parser (Price calculation (And possibly write a table for storing item data for a kill in?)
- [x] Populate corporation and character table using Resque when a mail is populated
- [x] Tell the zKB / Stomp / API fetchers to tell the Killmail parser about the new kill having been added

For the old manually posted killmails there are quite a lot more that needs to be done
- [ ] Update translations (Russian, German, French, Chinese and several others, possibly use the translation table)
- [ ] Convert old style killmails into the new format
- [ ] Look into calculating the crest hash and try and verify the mail, unless CCP has closed that hole like they said they would..

**Comment by karbowiak** [comment]
Market data population was added with 1fef23d78aa7386a8ddb116b5ff8f75090f84e80

**Comment by karbowiak** [comment]
Points calculation was added with f5808da3820832c19435027a04285bbe1934b3b1

**Comment by karbowiak** [comment]
Done, Most likely won't do manual mails, will reopen if need be
